prometheus_api_client/prometheus_connect.py

```diff
@@ -349,3 +351,43 @@ def custom_query_range(self, query: str, start_time: datetime, end_time: datetim
             )
 
         return data
+
+    @retry(stop_max_attempt_number=MAX_REQUEST_RETRIES, wait_fixed=CONNECTION_RETRY_WAIT_TIME)
+    def get_metric_aggregation(self, query, operations):
```

Comment:
hey @amitsagtani97, this looks like a very useful function :smile:  but I'm a bit unsure if we really need to implement a new class (`MetricAggregation`) for computing aggregations. Is it not possible to achieve the same functionality by making the `values` array an `np.array` instead of a `list`, and directly calling `numpy`'s min,max,etc functions on it?

Or did you have a specific use case in mind where `numpy`'s aggregtions won't be sufficient and we'll need a dedicated class?

Reply:
Hi @chauhankaranraj, yes we can remove the new class and perform operations within the prometheus_connect class. 

No, I didn't thought of any special cases at the moment and numpy aggregations looks good for our use.

Making the changes :+1: